shell/platform/darwin/ios/framework/Headers/FlutterDartRunner.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERDARTRUNNER_H_
+#define FLUTTER_FLUTTERDARTRUNNER_H_
+
+#import <Foundation/Foundation.h>
+
+#include "FlutterDartProject.h"
+#include "FlutterMacros.h"
+
+FLUTTER_EXPORT
+@interface FlutterDartRunner : NSObject
+
+- (instancetype)init;
+
+- (void)run:(NSString*)entrypoint;
```

Comment:
The signature `-(void) runWithEntrypoint:(NSString *) entrypoint;` is more idiomatic Objective-C. Also, since this is in a public header, API docs would be appreciated with a brief discussion of its use.

Reply:
Done